{
  "Header": {
    "Original": "CVE-2020-24659 (An issue was discovered in GnuTLS before 3.6.15. A server can trigger  ...)",
    "Line": 200942,
    "ID": "CVE-2020-24659",
    "Description": "(An issue was discovered in GnuTLS before 3.6.15. A server can trigger  ...)"
  },
  "Annotations": [
    {
      "Original": "- gnutls28 3.6.15-1 (bug #969547)",
      "Line": 200943,
      "Type": "package",
      "Package": "gnutls28",
      "Kind": "fixed",
      "Version": "3.6.15-1",
      "BugNo": 969547
    },
    {
      "Original": "[buster] - gnutls28 3.6.7-4+deb10u7",
      "Line": 200944,
      "Type": "package",
      "Release": "buster",
      "Package": "gnutls28",
      "Kind": "fixed",
      "Version": "3.6.7-4+deb10u7"
    },
    {
      "Original": "[stretch] - gnutls28 \u003cnot-affected\u003e (Vulnerable code introduced later)",
      "Line": 200945,
      "Type": "package",
      "Release": "stretch",
      "Package": "gnutls28",
      "Kind": "not-affected",
      "Description": "Vulnerable code introduced later"
    },
    {
      "Original": "NOTE: https://www.gnutls.org/security-new.html#GNUTLS-SA-2020-09-04",
      "Line": 200946,
      "Type": "NOTE",
      "Description": "https://www.gnutls.org/security-new.html#GNUTLS-SA-2020-09-04"
    },
    {
      "Original": "NOTE: https://gitlab.com/gnutls/gnutls/-/issues/1071",
      "Line": 200947,
      "Type": "NOTE",
      "Description": "https://gitlab.com/gnutls/gnutls/-/issues/1071"
    },
    {
      "Original": "NOTE: https://gitlab.com/gnutls/gnutls/-/commit/29ee67c205855e848a0a26e6d0e4f65b6b943e0a",
      "Line": 200948,
      "Type": "NOTE",
      "Description": "https://gitlab.com/gnutls/gnutls/-/commit/29ee67c205855e848a0a26e6d0e4f65b6b943e0a"
    }
  ]
}